src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -117,6 +117,14 @@ function refinementCommandCancellationFn(cmd) {
         return false;
     }
 
+    // There's conflict between with two elevation layers
```

Comment:
You could reword this comment:
```js
// Cancel the command if the tile already has a better texture.
// This is only needed for elevation layers, because we may have several
// concurrent layers but we can only use one texture.
```

Reply:
ok, thanks